libindy/src/api/anoncreds.rs

```diff
@@ -685,7 +691,7 @@ pub extern fn indy_prover_get_claims_for_proof_req(command_handle: i32,
 ///         "value": requested value of attribute
 ///         "restrictions": [
 ///             {
-///                 "schema_seq_no": int, (Optional)
+///                 "schema_key": {name (Optional), version (Optional), did (Optional)}, (Optional)
```

Comment:
Please add a comment about pattern matching for requested attributes in proof request (that it's case insensitive and ignore spaces). Add the comment in all methods dealing with proof request and pattern matching